typescript/examples/langchain-cdp-chatbot/chatbot.ts

```diff
@@ -87,8 +87,10 @@ async function initializeAgent() {
       apiKeyPrivateKey: process.env.CDP_API_KEY_PRIVATE_KEY?.replace(/\\n/g, "\n"),
       cdpWalletData: walletDataStr || undefined,
       networkId: process.env.NETWORK_ID || "base-sepolia",
+      gas: {
+        gasLimitMultiplier: 1.2,
+      },
```

Comment:
do we want to keep this config in the example? if so, let's use a different value since we now default to 1.2

Reply:
Good callout. I think we should remove it. It's optional, its in the docs, and uses are going through the examples to get a quick start.